CMakeLists.txt

```diff
@@ -255,6 +266,19 @@ else()
     set(MIOPEN_USE_MIOPENGEMM 0)
 endif()
 
+if(MIOPEN_USE_COMGR)
+    find_package(amd_comgr CONFIG)
+    if(amd_comgr_FOUND)
+        message(STATUS "Build with comgr ${amd_comgr_VERSION}")
+        set(MIOPEN_USE_COMGR On)
```

Comment:
This is pointless has it has already been set to `On`.

Reply:
Removed.